inc/thorvg_capi.h

```diff
@@ -68,6 +68,11 @@ typedef enum {
 } Tvg_Stroke_Fill;
 
 
+typedef enum {
+    TVG_FILL_RULE_WINDING = 0,
+    TVG_FILL_RULE_EVEN_ODD = 1
```

Comment:
TVG_FILL_RULE_EVEN_ODD = 1 -> 
TVG_FILL_RULE_EVEN_ODD 

No need to specify number of next item.